packages/cli/src/commands/init/init.ts

```diff
@@ -2,7 +2,7 @@ import os from 'os';
 import path from 'path';
 import fs from 'fs-extra';
 import minimist from 'minimist';
-import ora from 'ora';
+import {Ora} from 'ora';
```

Comment:
I don't think this change was required while bumping to 6.x, was it? I personally have no objections to either convention, but I was confused while reviewing the PR.

@thymikee please lmk if you're okay with that change.

Reply:
I believe it's due to a change in https://github.com/sindresorhus/ora/releases/tag/v6.0.0 where the ora export was changed and is no longer a ts namespace. But I may be mistaken.
I'll try once more without this change and verify whether or not I got any errors.

Reply:
I'm all good, it's the API change

Reply:
Thanks for pointing this out!